db/sql/0_msar.sql

```diff
@@ -956,7 +981,8 @@ SELECT array_agg(
     -- set the default value for the duplicate column if specified
     CASE WHEN copy_defaults THEN pg_get_expr(adbin, tab_id) END,
     -- We don't set a duplicate column as a primary key, since that would cause an error.
-    false
+    false,
+    col_description(tab_id, pg_columns.attnum)
```

Comment:
We should consider eventually making this configurable (it might be nice to have a comment like "Copy of Column X" by default here. However, let's leave it like you've got it for this PR.